spec/model/comment_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @user = User.new(id: 1, Name: 'Kenny', Bio: 'I am the greatest being alive', PostCounter: 1,
+                     Photo: 'https://google.com/googleIcon.png')
```

Comment:
- [OPTIONAL] Well done writing your validations tests so far. However, you're missing a small thing to do for the tests to be valid. Kindly make all Model specs (for `User`, `Post`, `Comment`, and `Like` model) has **Validation** tests as required in this milestone; See the example down below!
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```